### DISCUSSION TITLE
Deezer Issues

### DISCUSSION THREAD

**Discussion by SirPlancake** [TopiSenpai/LavaSrc#138] [Q&A] [answered]
When attempting to play a [song](https://deezer.page.link/o9pErRfQJAWAXf7e6) from deezer, it seems to be using some sort of a layer mpeg. This happens on every stable version and the beta versions, I have not tried any other. I'm also using lavalink v4.

If you want to have a little look, heres the sourcebin. https://sourceb.in/qFCMgT1sRM

My application.yml is also here: https://sourceb.in/d9QopYvMHY

Also a question, do you have a discord server?

**Comment by topi314** [comment, marked as answer]
Duplicate of https://github.com/topi314/LavaSrc/discussions/118

And there is no specific discord server
